core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -174,9 +174,9 @@ public boolean isWrite() {
      * existing inodes: /a
      * last edge: a->b
      *
-     * @return whether the last edge should be write locked
+     * @return whether the last edge should be locked (either read or write lock)
```

Comment:
if this includes read, why does the comparison not check for `READ`?

Reply:
I just removed the method, PTAL